api/Pathmaker/Pathmaker.Infrastructure/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -48,9 +48,9 @@ public static class IServiceCollectionExtensions {
         services.AddHangfireServer();
     }
 
-    private static void UseHangfire(this IApplicationBuilder builder) {
+    private static void UseHangfireDashboard(this IApplicationBuilder builder) {
         builder.UseRouting();
-        builder.UseHangfireDashboard();
+        HangfireApplicationBuilderExtensions.UseHangfireDashboard(builder);
```

Comment:
I guess there is a conflict name. Maybe rename your method to UseDefaultHangfireDashbaord